### PR TITLE
[EnivronmentParser] WiP and initial implementation for parsing and mergi...

### PR DESCRIPTION
**Pull request by k0nserv**
...ng the travis environment

### CODE REVIEW

File: lib/environment_parser.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    def initialize(env, *args)
+      @env = env
+      @env = [@env] if @env.is_a? String
+      @override_envs = args
+      @override_envs ||= []
+    end
+
+    # The resulting environment has after merging
+    # the environment from the travis file with
+    # the specified overrides. This method is lacy
```
Comment (orta) [inline lib/environment_parser.rb]: lazy

Comment (k0nserv) [inline lib/environment_parser.rb]: ouch :)

**Comment by orta** [comment]
I cannot merge this

**Comment by k0nserv** [comment]
Now you can :)

**Comment by orta** [comment]
ace, let's do it.

**Pull request merged**